src/server/server.rs

```diff
@@ -62,18 +77,27 @@ impl<T: RaftStoreRouter, S: StoreAddrResolver + 'static> Server<T, S> {
         debug_engines: Option<Engines>,
         import_service: Option<ImportSSTService<T>>,
     ) -> Result<Self> {
+        // A helper thread (or pool) for transport layer.
+        let mut pool_builder = thread_pool::Builder::new();
```

Comment:
I suggest to move it into `RuntimeBuilder` so that it can have a clean interface.